### PR TITLE
fixed entityManager initialization during CF readout

### PR DESCRIPTION
**Pull request by markussc**
Also introduces doctrine ManagerRegistry injection into services

### CODE REVIEW

File: src/Entity/EntityCustomField.php [hunk 1]
```diff
@@ -140,7 +140,7 @@ private function getEntityData()
         if ($this->entityValue && $this->entityValue['entityClass']) {
             // TODO: find a better way to retrieve the entity manager
             global $kernel;
-            $em = $kernel->getContainer()->getDoctrine_Orm_DefaultEntityManagerService();
+            $em = $kernel->getContainer()->get('doctrine');
```
Comment (SimonHeimberg) [inline src/Entity/EntityCustomField.php]: $em = ...->get('doctrine')->getManager();

**Comment by SimonHeimberg** [comment]
Sorry, I do not find the unit tests in pa for checking this. Where are they?

**Comment by markussc** [comment]
@SimonHeimberg : do we have unit tests for CF? not sure.

**Comment by SimonHeimberg** [comment]
When I remember right, you told about many tests which showed many failures when this bundle was new.

**Pull request merged**